src/styling-panel-definition.js

```diff
@@ -77,6 +80,191 @@ export const getStyleEditorDefinition = () => ({
+            MediaLibrary: {
+              component: 'media-library-button',
+              ref: 'style.background.url',
+              translation: 'MediaLibrary.Header',
+              show: (data) => propertyResolver.getValue(data, 'style.background.mode') === 'media',
+            },
+            imageSize: {
+              component: 'dropdown',
+              ref: 'style.background.size',
+              defaultValue: styleDefaults.BACKGROUND_SIZE,
+              options: [
+                {
+                  value: 'auto',
+                  translation: 'properties.backgroundImage.originalSize',
+                },
+                {
+                  value: 'alwaysFit',
+                  translation: 'properties.backgroundImage.sizeAlwaysFit',
+                },
+                {
+                  value: 'fitWidth',
+                  translation: 'properties.backgroundImage.sizeFitWidth',
+                },
+                {
+                  value: 'fitHeight',
+                  translation: 'properties.backgroundImage.sizeFitHeight',
+                },
+                {
+                  value: 'stretchFit',
+                  translation: 'properties.backgroundImage.sizeStretch',
+                },
+                {
+                  value: 'alwaysFill',
+                  translation: 'properties.backgroundImage.sizeAlwaysFill',
+                },
+              ],
+              change: (data) => {
+                let currentPosition = propertyResolver.getValue(data, 'style.background.position');
+                if (currentPosition) currentPosition = styleDefaults.BGIMAGE_POSITION;
+              },
+              show: (data) =>
+                propertyResolver.getValue(data, 'style.background.mode') === 'media' &&
+                !!propertyResolver.getValue(data, 'style.background.url.qStaticContentUrlDef.qUrl'),
+            },
+            position: {
+              component: 'position-grid',
+              ref: 'style.background.position',
+              translation: 'properties.backgroundImage.position',
+              defaultValue: styleDefaults.BGIMAGE_POSITION,
+              show: (data) =>
+                propertyResolver.getValue(data, 'style.background.mode') === 'media' &&
+                propertyResolver.getValue(data, 'style.background.url.qStaticContentUrlDef.qUrl') &&
+                propertyResolver.getValue(data, 'style.background.size') !== 'stretchFit',
+              currentSizeRef: 'style.background.size',
+            },
+          },
+        },
+      },
+    },
+    backgroundBorder: {
+      component: 'panel-section',
+      translation: 'properties.border',
+      items: {
+        backgroundBorderItem: {
+          items: {
+            useBorder: {
+              type: 'boolean',
+              component: 'switch',
+              ref: 'style.border.useBorder',
+              translation: 'properties.border.use',
+              defaultValue: toggleOptions[1].value,
```

Comment:
Just `false` would be easier I think

Reply:
Oh weird, I had fixed some details before but they are still here... I'll fix it.